packages/website/src/features/Packages/FunctionOutput.tsx

```diff
@@ -82,11 +86,19 @@ export const FunctionOutput: FC<{
           </Text>
         );
       } else if (isArray(value)) {
-        return value.map((val, idx) => (
-          <Text fontSize="xs" display="block" key={idx}>
-            {String(val)}
-          </Text>
-        ));
+        if (index !== undefined) {
+          return (
+            <Text fontSize="xs" display="block">
+              {String(value[index])}
+            </Text>
+          );
+        } else {
+          return value.map((val, idx) => (
+            <Text fontSize="xs" display="block" key={idx}>
+              {index !== undefined ? String(val[index]) : String(val)}
```

Comment:
In this case, `index !== undefined` is giving always `false`, right?

Reply:
Hey @mjlescano ! you're right! thanks for catching this one